tests/unit/conftest.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+
+from expertsystem import ui
+from expertsystem.data import ParticleCollection
+from expertsystem.state.particle import DATABASE
+
+
+@pytest.fixture(scope="module")
```

Comment:
Interesting, so because pytest is collecting all tests, this will also be loaded automatically right? I guess this will not allow to test `test_particle.py` individually. I assume some kind of import will be needed.

Btw, I think the scope of this can be set to `package`

Reply:
Yes, see
https://docs.pytest.org/en/stable/fixture.html#conftest-py-sharing-fixture-functions

What I find more remarkable is that `mypy`, `pylint`, etc somehow understand this shared fixture without an `import` statement.

> Btw, I think the scope of this can be set to package

See a677a67a9fae11e06c749be76652a4026b1a3a31